content/en/docs/concepts/data-collection.md

```diff
@@ -41,11 +45,180 @@ The Collector is made up of the following components:
+
+You can set multiple protocols for a single receiver and make them listen to different ports by default. 
+The table below shows supported receiver formats for each signal type:
+
+| Signal Source        | Traces             | Metrics            | Logs               |
+| :---                 |    :----:          |  :---:             |               ---: |
+| Host Metrics         |                    | :heavy_check_mark: |                    |
+| Jaeger               | :heavy_check_mark: |                    |                    |
+| Kafka                | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| OpenCensus           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| OpenTelemetry (OTLP) | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Prometheus           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| Zipkin               | :heavy_check_mark: |                    |                    |
+
+Learn how to configure receivers in the [configuration documentation](/docs/collector/configuration/#receivers).
+
+### Processors
+The job of the processor is to filter unwanted telemetry data and inject additional attributes to the data 
+before it is sent to the exporter. While receivers and exporters, the capabilities of processors differ immensely
+from one processor to the other. Processors are run on data between being received and being exported.
+While processors are optional, [some are
+recommended](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor#recommended-processors).
+
+The table below shows some currently supported processors, and the signals they
+possess:
+
+| Signal                 |       Traces        |      Metrics       |               Logs |
+|:-----------------------|:-------------------:|:------------------:|-------------------:|
+| Attributes             | :heavy_check_mark:  |                    | :heavy_check_mark: |
+| Batch                  | :heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark: |
+| Filter                 |                     | :heavy_check_mark: |                    |
+| Memory Limiter         | :heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark: |
+| Probabilistic Sampling | :heavy_check_mark:  |                    |                    |
+| Resource               | :heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark: |
+| Span                   | :heavy_check_mark:  |                    |                    |
+
+Processors are optional, although [some are 
+recommended](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor#recommended-processors). 
+
+Processors can be configured to export data into or consolidate data coming in from 
+multiple systems, where different names are used to represent the same data. It is important to learn
+about the types of processors in detail to maximize their usage.
```

Comment:
> It is important to learn
about the types of processors in detail to maximize their usage.

Hmmmm. Is it? I mean, I think it is, but I'm not sure about this sentence. It's perfectly valid for lots of people to just stand up a "pass through" collector just so that they can manage secrets for a backend in one place. I wouldn't feel comfortable making a _best practices_ statement like this just yet, not at least without consulting several people in the community.